plugins/pci-psp/meson.build

```diff
@@ -14,4 +14,5 @@ plugin_builtins += static_library('fu_plugin_pci_psp',
   dependencies: plugin_deps,
 )
 umockdev_tests += files('pci_psp_test.py')
+enumeration_data += files('tests/strix.json')
```

Comment:
I think this should be called something like `amd-strix.json` to namespace it a bit. We also need an emulation.json file so this runs in CI -- e.g. like we have in `plugins/dell-kestrel/tests/dell-kestrel-enumerate.json`